modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1704,8 +1704,17 @@ This command does not take any options, except for the default ones: `--help`, `
 > runtime-mgr standalone-application list [options]
 ----
 
-This command lists all standalone (on-premises) applications. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command lists all standalone (on-premises) applications.
+
+Besides the default `--help`, `-f`/`--fields` and `-o`/`--output` options, this command also takes:
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+|Value |Description
+| `--limit <num>` | Number of results to retrieve
+| `--offset`      | Offsets the number of applications passed
```

Comment:
```suggestion
| `--offset <num>`      | Offsets the number of applications passed
```

Adding `<num>` as part of offset.

Also some commands support `--sort <fieldName>`